packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts

```diff
@@ -98,5 +168,163 @@ export async function forceLightningLwcPreview(sourceUri: vscode.Uri) {
 
     await commandlet.run();
     telemetryService.sendCommandEvent(logName, startTime);
+  } else if (desktopSelected) {
+    try {
+      await openBrowser(fullUrl);
+      telemetryService.sendCommandEvent(logName, startTime);
+    } catch (e) {
+      showError(e, logName, commandName);
+    }
+  }
+}
+
+async function selectPlatformAndExecute(
+  fullUrl: string,
+  startTime: [number, number],
+  componentName: string
+) {
+  const platformSelection = await vscode.window.showQuickPick(platformOptions, {
+    placeHolder: nls.localize('force_lightning_lwc_platform_selection')
+  });
+  if (!platformSelection) {
+    vscode.window.showWarningMessage(
+      nls.localize('force_lightning_lwc_cancelled')
+    );
+    return;
+  }
+
+  const desktopSelected = platformSelection.id === PreviewPlatformType.Desktop;
```

Comment:
nit: can we tweak the naming a bit here just so that it's more readable
```js
const isDesktop = platformSelection.id === PreviewPlatformType.Desktop;
const isAndroid = platformSelection.id === PreviewPlatformType.Android;
```